CMakeLists.txt

```diff
@@ -223,7 +223,7 @@ function(compile_flatbuffers_schema_to_cpp SRC_FBS)
   add_custom_command(
     OUTPUT ${GEN_HEADER}
     COMMAND "${FLATBUFFERS_FLATC_EXECUTABLE}" -c --no-includes --gen-mutable
-            --gen-object-api -o "${SRC_FBS_DIR}"
+            --gen-object-api --gen-compare -o "${SRC_FBS_DIR}"
```

Comment:
why is this by default on?

Reply:
"compile_flatbuffers_schema_to_cpp" is used to compile the schemas for the tests. It is not used anywhere else. I have to enable --gen-compare in order to successfully compile the tests

Reply:
It may be used by others, but I suppose this is ok.